hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -79,6 +79,11 @@
 
   private static final PathFilter ALLOW_ALL_FILTER = file -> true;
 
+  private static final Set<String> DATA_FILE_EXTENSIONS = Arrays.stream(HoodieFileFormat.values())
```

Comment:
Should this be BASE_DATA_FILE_EXTENSION ?

Reply:
renamed

Reply:
For naming we stick to `base file` over data file. the last of the data file references have a PR open to remove. So kindly rename to `BASE_FILE_EXTENSIONS` and can this be in `HoodieFileFormat` itself ?